ambry-frontend/src/main/java/com/github/ambry/frontend/AmbrySecurityService.java

```diff
@@ -269,6 +270,9 @@ public void processResponse(RestRequest restRequest, RestResponseChannel respons
                   new Date(blobInfo.getBlobProperties().getCreationTimeInMs()));
             }
             break;
+          case DELETE:
```

Comment:
Just noting that this might also require similar change in internal Ambry code (AmbryLiSecurityService). You might already be aware :)

Reply:
for sure